markdown/assignments/pool_concept/tasks/swing.md

```diff
@@ -0,0 +1,31 @@
+In den Vorgaben finden Sie die Implementierung für ein TicTacToe-Spiel.
+Ihre Aufgabe ist es, eine grafische Benutzeroberfläche für das Spiel zu entwickeln.
+
+Ihr Fenster soll sich immer in der Mitte des Bildschirms starten und einen Titel besitzen.
```

Comment:
öhm, wie geht das mit "immer in der Mitte des Bildschirms"? ist das noch swing oder eher betriebssystem?

Reply:
swing macht das. Man kann die Position des Fensters bestimmen. Ich verrate mal nicht die Antwort ^^

Reply:
ja ok. dann muss ich hier wohl nochmal in der musterlösung schmökern und ggf. die vorlesung ergänzen :)